client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue

```diff
@@ -382,10 +472,26 @@ export default {
       this.localStatement = JSON.parse(JSON.stringify(this.statement))
       this.localStatement.attributes.authoredDate = this.convertDate(this.localStatement.attributes.authoredDate)
       this.localStatement.attributes.submitDate = this.convertDate(this.localStatement.attributes.submitDate)
+
+      this.finalMailDefaultText = Translator.trans('statement.send.final_mail.default', {
+        hasStatementText: this.localStatement.attributes.fullText.length < 2000 ? 0 : 1,
+        orgaName: this.procedure.orgaName,
+        procedureName: this.procedure.name,
+        statementText: this.localStatement.attributes.fullText,
+        statementRecommendation: this.localStatement.attributes.recommendation
+      })
     },
 
     syncAuthorAndSubmitter () {
       this.localStatement.attributes.submitName = this.localStatement.attributes.authorName
+    },
+
+    updateCounties (val) {
+      console.log(val, this.localStatement.attributes.counties)
+    },
+
+    updateMunicipalities (val) {
+      console.log(val, this.localStatement.attributes.municipaities)
```

Comment:
Add update logic here, old logic is here client/js/components/statement/assessmentTable/DetailView/DetailView.vue:192